symfony/plugins/orangehrmPerformancePlugin/Api/Model/PerformanceTrackReviewerModel.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+
+namespace OrangeHRM\Performance\Api\Model;
+
+use OrangeHRM\Core\Api\V2\Serializer\ModelTrait;
+use OrangeHRM\Core\Api\V2\Serializer\Normalizable;
+use OrangeHRM\Entity\PerformanceTrackerReviewer;
+
+/**
+ *
+ */
+class PerformanceTrackReviewerModel implements Normalizable
+{
+    use ModelTrait;
+
+    public function __construct(PerformanceTrackerReviewer $performanceTrackerReviewer)
+    {
+        $this->setEntity($performanceTrackerReviewer);
+        $this->setFilters(
+            [
+                'id',
+                ['getReviewer', 'getEmpNumber'],
+                ['getReviewer', 'getFirstName'],
+                ['getReviewer', 'getLastName'],
+            ]
+        );
+        $this->setAttributeNames(
+            [
+                'id',
+                [ 'empNumber'],
+                [ 'firstName'],
+                [ 'lastName'],
+            ]
+        );
```

Comment:
```suggestion
        $this->setFilters(
            [
                'id',
                ['reviewer', 'empNumber'],
                ['reviewer', 'lastName'],
                ['reviewer', 'firstName'],
                ['reviewer', 'middleName'],
                ['reviewer', 'employeeId'],
                ['reviewer', 'getEmployeeTerminationRecord', 'getId'],
            ]
        );
        $this->setAttributeNames(
            [
                'id',
                'empNumber',
                'lastName',
                'firstName',
                'middleName',
                'employeeId',
                'terminationId',
            ]
        );
```

Reply:
```suggestion
        $this->setFilters(
            [
                'id',
                ['getReviewer', 'getEmpNumber'],
                ['getReviewer', 'getFirstName'],
                ['getReviewer', 'getLastName'],
            ]
        );
        $this->setAttributeNames(
            [
                'id',
                [ 'empNumber'],
                [ 'firstName'],
                [ 'lastName'],
            ]
        );
```
$this->setFilters(
           [
                'id',
                ['getReviewer', 'getEmpNumber'],
                ['getReviewer', 'getLastName'],
                ['getReviewer', 'getFirstName'],
                ['getReviewer', 'getMiddleName'],
                ['getReviewer', 'getEmployeeId'],
                ['getReviewer', 'getEmployeeTerminationRecord', 'getId'],
            ]
        );

Reply:
set it to this way to resolve an error when calling to Reviewer